### DISCUSSION TITLE
Dx Cluster : add sh/dx or sh/mydx listings to SPOTS display

### DISCUSSION THREAD

**Discussion by rsaue** [foldynl/QLog#286] [Ideas]
Hi,
Unless I have overseen something in the setup, QLog DX Cluster window SPOT WIEW today only shows spots that is received starting with "DX de": 

             DX de W3OA-#:    14029.0  VU2TMP       CW  8 dB 27 WPM CQ             1658Z

When logging into a cluster I usually do a listing of the last xx spots e.g. SH/DX 50 or SH/MYDX 50. In CC-cluster which my cluster node uses, each listed spot comes without the "DX de" prefix: 

              14222.0  9L5M        04-Dec-2023 1655Z  TNX                       <EB1DJ>
               7007.0  F8FSN       04-Dec-2023 1655Z                            <IU5KZF>
               7019.8  PA3AAV      04-Dec-2023 1656Z  OK1WC Memorial            <IT9VDQ>
               14071.3  VK9CY       04-Dec-2023 1656Z  Tnx 2.bnd, Vy 73 Gl Dx    <DM2DMI>

It looks like the dxspider software used by hamqth.com display sh/dx listings the same way. 

These sports are not shown in the QLog SPOTS window. 

I suggest you make such spots available in the SPOTS window. 

73 de LB2EG Richard

**Comment by foldynl** [comment]
Yes, that's true. QLog shows only spots with leading "DX". 

I will think about it
